src/msg.c

```diff
@@ -3693,27 +3692,48 @@ void msg_reset_gfx_state(void) {
     gSPDisplayList(gMainGfxPos++, D_8014C500);
 }
 
-#if VERSION_IQUE
+#if VERSION_IQUE && !defined(NON_MATCHING)
 INCLUDE_ASM(s32, "msg", msg_draw_char);
 #else
 void msg_draw_char(MessagePrintState* printer, MessageDrawState* drawState, s32 charIndex, s32 palette, s32 posX, s32 posY) {
-    MessageCharset* messageCharset = MsgCharsets[drawState->font];
-    s32 fontVariant = drawState->fontVariant;
+    MessageCharset* messageCharset;
+    s32 fontVariant;
 
-    s32 clipUly = drawState->clipY[0];
-    s32 clipLry = drawState->clipY[1];
-    s32 clipUlx = drawState->clipX[0];
-    s32 clipLrx = drawState->clipX[1];
+    s32 clipUly;
+    s32 clipLry;
+    s32 clipUlx;
+    s32 clipLrx;
 
-    s32 rightPosX = posX + (s32)(drawState->charScale.x * messageCharset->texSize.x);
-    s32 rightPosY = posY + (s32)(drawState->charScale.y * messageCharset->texSize.y);
+    s32 rightPosX;
+    s32 rightPosY;
 
     f32 clipOffset;
     s32 texOffsetX;
     s32 texOffsetY;
     s32 ulx, uly, lrx, lry;
     s32 dsdx, dtdy;
 
+#if VERSION_IQUE
+    if (charIndex == 0x33F) {
+        load_font_data(charset_standard_OFFSET + 0x19F80, 0x80, D_801544A0);
```

Comment:
- Can we get a label for what the 0x19F80 points to? (@ethteck)
- `0x80` should probably be a `sizeof(D_801544A0[0])`
- `D_801544A0` should probably be `D_801544A0[0]`

Reply:
- let him decide
- **done**
- **done**

Reply:
uhh, no clue what that is.. I think a todo suffices until we know more, but maybe @HailSanta knows?

Reply:
> uhh, no clue what that is.. I think a todo suffices until we know more, but maybe @HailSanta knows?

FYI, that's the offset for image char 勋  `(16 * 16 / 2) * 0x33F`. Can't add an address offset in it because I can't resplit the standard charset in splat.yaml:
```yaml
    - [0x10E120, linker_offset,       standard]
    - [0x10E120, pm_charset,          standard, 16, 16, 0x9E0] # <-- resplit possible?
 ```